fix(content): Remnant: Purging contractions from Remnant text (#7948)
endless-sky/endless-sky: data/remnant/remnant.txt

phrase "remnant uncontacted hostile"
	word
		"(They sent you hail consisting of a sign language you do not understand, and it does not seem friendly.)"

> 		"(They sign so quickly that your hail receiver can not process it all.)"

Reviewer: ```suggestion
		"(They sign so quickly that your hail receiver cannot process it all.)"
```
